jakartaee-api/pom.xml

```diff
@@ -78,6 +87,15 @@
             <version>${project.version}</version>
             <optional>false</optional>
         </dependency>
+        <!-- Core Profile API -->
+        <!-- We specify it here directly, so it's "closer" to the user app.
+             This way it will win in Maven's resolution mechanism -->
+        <dependency>
+            <groupId>jakarta.platform</groupId>
+            <artifactId>jakarta.jakartaee-core-api</artifactId>
+            <version>${project.version}</version>
+            <optional>false</optional>
```

Comment:
@JanWesterkamp-iJUG questions the correctness of having this here. Remove the `<optional>false</optional>`.

Reply:
Looks like you catched all the `<optional>false</optional>` statements so far.